src/scss/_dependencies.scss

```diff
@@ -65,7 +65,9 @@
  *
  * This is where the majority of our day-to-day styling will take place.  Component styles should be prefixed with `.c-`
  */
-@import 'components/cards.scss';
+@import 'components/buttons';
```

Comment:
Didn’t pick up on this before, but buttons should probably be an object (so `o-`) if we are using the definition of objects can be assigned to a tag.

It’s a bit subjective, so don’t particularly mind either way. @DamianMullins any thoughts?

Reply:
😄  I actually had made them an object first before changing to a component. I wasn't sure since some of them are a little more complex e.g. `btnToggle`

Reply:
Just had a look in the UK codebase and they are defined as objects there, I think we should follow their lead and go with `object`?

Reply:
Yep – agree